### PR TITLE
fix: body scroll lock

### PR DESCRIPTION
**Pull request by RakowskiPrzemyslaw**
### Related Issues

closes https://github.com/vuestorefront/vue-storefront/issues/5514

### Short Description and Why It's Useful

Removing ` overflow-x: hidden` from html tag fixes functionality of body-scroll-lock and hides scrollbar

### Screenshots of Visual Changes before/after (if There Are Any)

image

### Which Environment This Relates To
Check your case. In case of any doubts please read about [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html)

- [ ] Test version (https://test.storefrontcloud.io) - this is a new feature or improvement for Vue Storefront. I've created branch from `develop` branch and want to merge it back to `develop`
- [x] RC version (https://next.storefrontcloud.io) - this is a stabilisation fix for Release Candidate of Vue Storefront. I've created branch from `release` branch and want to merge it back to `release`
- [ ] Stable version (https://demo.storefrontcloud.io) - this is an important fix for current stable version. I've created branch from `hotfix` or `master` branch and want to merge it back to `hotfix`

### Upgrade Notes and Changelog
- [x] No upgrade steps required (100% backward compatibility and no breaking changes)
- [ ] I've updated the [Upgrade notes](https://github.com/DivanteLtd/vue-storefront/blob/develop/docs/guide/upgrade-notes/README.md) and [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing Vue Storefront sites with this new feature
- [x] (Next only) I've followed this [instruction](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and I've created a `.js` file with information about my Pull Request

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vue-storefront/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: packages/core/docs/changelog/5514.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  description: 'removed  overflow-x: hidden from html tag',
```
Comment (Fifciu) [inline packages/core/docs/changelog/5514.js:2]: Please get rid of double space

**Review by Fifciu — CHANGES_REQUESTED** [review]
Looks fine, just fix this double space

File: packages/core/docs/changelog/5514.js (hunk 1 above)
Comment (RakowskiPrzemyslaw) [reply packages/core/docs/changelog/5514.js]: good catch, removed

**Review by filrak — APPROVED** [review]

**Comment by Baroshem** [comment]
@RakowskiPrzemyslaw 

This pull request fixed a problem with checkout body scroll but added a global bug with mobile view Filip discovered yesterday. 

To solve current issue, different solution must be created and tested on all devices.

**Comment by RakowskiPrzemyslaw** [comment]
@Baroshem maybe just wrapping it in media query would do the trick? What do you think?

**Comment by Baroshem** [comment]
@RakowskiPrzemyslaw 
Yeah, it seems to bo working -> https://github.com/vuestorefront/vue-storefront/pull/5682/commits/fec4a5f7dac8f352b2261aa316bf611e04418587

**Pull request merged**